aim/web/ui/src/pages/Metrics/components/MetricsBar/MetricsBar.tsx

```diff
@@ -118,6 +120,9 @@ function MetricsBar({
                 href='https://github.com/aimhubio/aim#searching-experiments'
```

Comment:
@gorarakelyan, what should be the current link for the `Searching Experiments` section in the docs, or do we need this link here at all?

Reply:
QL basics @roubkar, but that is in progress

Reply:
Added https://aimstack.readthedocs.io/en/latest/quick_start/QL_basics.html#searching-runs-and-metrics :)

Reply:
We're removing this link from the menu now, in the future will add a link to docs in sidebar

Reply:
@roubkar ok